scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -5,6 +5,7 @@ workflow GvsImportGenomes {
   input {
     Array[File] input_vcfs
     Array[File] input_vcf_indexes
+    Array[String] research_ids
```

Comment:
Maybe let's call this `external_sample_name` or something like that?  research_ids is pretty specific to AoU

Reply:
Also -- can you update the example json (GvsImportGenomes.example.inputs.json)